gradle.properties

```diff
@@ -2,14 +2,14 @@ projectVersion=2.0.1.BUILD-SNAPSHOT
 developers=Graeme Rocher
 githubBranch=master
 githubSlug=micronaut-projects/micronaut-cassandra
-groovyVersion=2.5.11
+groovyVersion=3.0.4
 micronautDocsVersion=1.0.23
 micronautBuildVersion=1.1.5
-micronautVersion=1.3.5
+micronautVersion=2.0.0
```

Comment:
Same question regarding Micronaut version. Can we stick to 1.3?

Reply:
changed to 1.3.5